.gitignore

```diff
@@ -18,3 +18,4 @@ Temporary Items
 /dist
 .vscode/
 yarn-error.log
+js/tempCodeRunnerFile.js
```

Comment:
what is this file?

Reply:
I think that is something created by VSC when I run my code...

Reply:
I've just checked that, this file was created automatically by running some part of my code in VSC. I will just remove it, it's redundant.